web/server/README.md

```diff
@@ -91,15 +115,57 @@ Netdata supports access lists in `netdata.conf`:
 
 - `allow management from` checks the IPs to allow API management calls. Management via the API is currently supported for [health](../api/health/#health-management-api)
 
+## SSL access list
+
+When the certificate is defined as one of the configuration to use, the Netdata will enable the TLS and it will redirect all HTTP request to HTTPS, on the other hand, the stream will be text pure until the user define an ACL to it, of course it is also possible to change the server behavior changing the ACL. For both cases we will use the variable `bind to` to change the default behavior.
+
+To change the behavior of master or slave we will use the definition `^SSL` that can have two possible values
```

Comment:
The slave config doesn't belong there. These settings tell the server which ports to listen to and how. You can have a scenario with host A streaming to host B streaming to host C. You can't tell a host how to behave as a client, in the same place where we tell it how to behave as a server. It's much simpler actually for the slaves. You will need to modify what can be added to the `destination` of stream.conf for the slave. i.e. this config param:
```
    # Where is the receiving netdata?
    # A space separated list of:
    #
    #      [PROTOCOL:]HOST[%INTERFACE][:PORT]
    #
    # If many are given, the first available will get the metrics.
    #
    # PROTOCOL  = tcp, udp, or unix (only tcp and unix are supported by masters)
    # HOST      = an IPv4, IPv6 IP, or a hostname, or a unix domain socket path.
    #             IPv6 IPs should be given with brackets [ip:address]
    # INTERFACE = the network interface to use (only for IPv6)
    # PORT      = the port number or service name (/etc/services)
    #
    # This communication is not HTTP (it cannot be proxied by web proxies).
    destination =
```
So we add an option here to tell the slave to communicate with the master via ssl.

I'll be testing with these settings for now, but it needs to change.

Reply:
All right, I understood. I am waiting you finish your tests, when it is done I will change what is necessary, including this documentation.

Reply:
I appended more a parameter for the destination, this way we split the configuration.